SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs

```diff
@@ -97,9 +99,9 @@ public async Task<SubscriptionInfo> GetLanguagePairs()
 			return await _httpClient.GetResult<SubscriptionInfo>(response);
 		}
 
-		public async Task<HttpResponseMessage> SendFeedback(SegmentId? segmentId, dynamic rating, string originalText, string improvement)
+		public async Task<HttpResponseMessage> SendFeedback(SegmentId? segmentId, dynamic rating, string originalText, string improvement, QualityEstimation qualityEstimation)
```

Comment:
Should we consider to use a object for all this properties something like "Feedbackmodel"? this method has to many parameters.

Reply:
done